operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -150,35 +150,50 @@ func (c *CentralComponentSpec) GetAdminPasswordGenerationDisabled() bool {
 
 // CentralDBEnabled returns a bool if CentralDBSpec is not nil
```

Comment:
Please update docstrings as you go. :wink:

Reply:
Did not really make sense to me to replicate the function logic in the comment, I changed it to something more generic.